src/main/resources/db/changelog/changes/07-create_shopping_carts_table.yaml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_shopping_carts_table
+      author: DEz
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
If column have `primaryKey: true`, `nullable: false` constraint can be not mentioned here

Reply:
also it's up to you, but `nullable: false` can be removed :)